cmd/zoekt/main.go

```diff
@@ -46,7 +46,7 @@ func displayMatches(files []zoekt.FileMatch, pat string, withRepo bool, list boo
 	}
 }
 
-func loadShard(fn string, verbose bool) (zoekt.Searcher, error) {
+func loadShard(fn string, verbose bool) (zoekt.MuxSearcher, error) {
```

Comment:
this file doesn't need to change. loadShard call sites just uses Searcher

Reply:
Not relevant anymore. New commit removes this part.